cartography/util.py

```diff
@@ -93,12 +93,11 @@ def merge_module_sync_metadata(
             n:SyncMetadata, n.firstseen=timestamp()
         SET n.syncedtype='${synced_type}',
             n.grouptype='${group_type}',
-            n.groupid={group_id},
-            n.lastupdated={UPDATE_TAG}
```

Comment:
why is this update_tag in a different syntax with group_id?

Reply:
Here I use `group_id` as a Template parameter, wrapped again inside quotes. There's also a integ test that covers this change.